lib/index.js

```diff
@@ -0,0 +1,3 @@
+// This is the base url
+// make requests with `herokuUrl + "api/v1/foods"` etc
+const herokuUrl = require("./heroku-url").herokuUrl
```

Comment:
I am not sure we need this in index.js since we won't be making API calls from there.

Reply:
Yeah, this isn't showing up in my text editor. I will delete these lines from `index.js`, I agree it doesn't belong in that file.